domain/schema/model/sql/0019-application.sql

```diff
@@ -279,3 +279,32 @@ SELECT
     a.application_uuid
 FROM application_endpoint AS a
 JOIN charm_relation AS c ON a.charm_relation_uuid = c.uuid;
+
+-- v_principal_subordinate returns the uuids of an subordinate application
+-- and its principal application. A principal application may multiple
+-- subordinates as a subordinate may be related to multiple principals.
```

Comment:
```suggestion
-- subordinates and a subordinate may be related to multiple principals.
```
The second clause doesn't imply the first here.